etc/byor_config.py

```diff
@@ -0,0 +1,6 @@
+c = get_config()
+load_subconfig('etc/base_config.py')
+load_subconfig('etc/github_auth.py')
+
+from dockerspawner import DockerSpawner
+c.DockerSpawner.hub_ip_connect = c.JupyterHub.hub_ip
```

Comment:
let's remove the last 2 lines. check that it doesn't break anything. if so, you can just load_subconfig('etc/local_config.py')